packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -210,10 +219,71 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static updateSelectedGtrOption(offer, optionName) {
+    let optionComfort = false;
+    offer.options.forEach((option) => {
+      if (
+        option.name.startsWith('gtr_') &&
+        option.selected !== null &&
+        optionName.startsWith('gtr_')
+      ) {
+        if (option.name !== optionName) {
+          set(option, 'selected', false);
+        }
+        if (optionName.match(/^gtr_\d{1,2}m_/) && option.selected === true) {
+          optionComfort = true;
+        }
+        set(offer, 'gtrComfortActivated', optionComfort);
+      }
+    });
+  }
+
+  updateOfferPriceAndGtr(value, offer, optionName) {
+    this.constructor.updateSelectedGtrOption(offer, optionName);
+    this.constructor.updateOfferDisplayedPrice(value, offer, optionName);
+  }
+
   static isChosen(option) {
     return option.choosedValue > 0;
   }
 
+  static isGtrOption(optionName) {
+    return optionName.startsWith('gtr_') ? 1 : 0;
```

Comment:
**question:** Ternary operation needed? Because `String.prototype.startsWith` already returned a boolean.

Reply:
no need of ?. I made the change